File: 2010/one/One_4.cs
using System;

class Test {
    public static int x = 3;
    public static int y;
    public int a = x;
    public int b;

    static Test () {
        x = 5;
    }

    public Test () {
        x++;
    }

    static void Main () {
        Test t1 = new Test();
        Test t2 = new Test();
        Console.Write(Test.x);
        Console.Write(' ');
        Console.Write(Test.y);
        Console.Write(' ');
        Console.Write(t2.a);
        Console.Write(' ');
        Console.Write(t2.b);
    }
}

File: 2013/one/One_4.java
class Test {
    public static void main ( String [] args ) {
    try {
        try {
            throw new Exception ();
        }
        catch ( RuntimeException e ) {
            System.out.println ("Inner runtime exception");
        }
        finally {
            System.out.println ("Finally");
        }
        try {
            ;
        }
        catch ( Exception e ){
            System.out.println ("Exception caught");
            throw new RuntimeException ();
        }
    } catch ( RuntimeException e ) {
        System.out.println ("Outer runtime exception");
    } catch ( Exception e ) {
        System.out.println ("Outer exception");
        throw new RuntimeException ();
        }
    }
}